forge-kernel/src/tools/forge_verify.cpp

```
@@ -0,0 +1,744 @@
+                    std::snprintf(buf, sizeof buf, "\\u%04x", c);
+                    out += buf;
+                } else {
+                    out.push_back(c);
+                }
+        }
+    }
+    return out;
+}
+
+// Value of a top-level "key": "..." string field. Returns false when absent.
+bool jsonString(const std::string& line, const std::string& key, std::string& out) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)